AlphaWalletAddress.podspec

```diff
@@ -20,7 +20,7 @@ Pod::Spec.new do |s|
   s.swift_version    = '4.2'
   s.platform         = :ios, "13.0"
   s.source           = { :git => 'git@github.com:AlphaWallet/alpha-wallet-ios.git', :tag => "#{s.version}" }
-  s.source_files     = 'AlphaWalletAddress/**/*.{h,m,swift}'
+  s.source_files     = 'Modules/AlphaWalletAddress/AlphaWalletAddress/**/*.{h,m,swift}'
```

Comment:
> Can you make these modules (lowercase "m") instead of Modules? I think the latter will break for case-sensitive file systems.

it should be possible, i'l try